src/System.IO.Compression/tests/ZipArchive/zip_ManualAndCompatibilityTests.cs

```diff
@@ -45,7 +47,7 @@ public class zip_ManualAndCompatabilityTests : ZipFileTestBase
         /// For example, the file "aa\bb\cc\dd" in a zip created on Unix should be one file "aa\bb\cc\dd" whereas the same file
         /// in a zip created on Windows should be interpreted as the file "dd" underneath three subdirectories.
         /// </summary>
-        [Theory]
+        [ConditionalTheory(nameof(IsUsingNewPathNormalization))]
```

Comment:
Oh I see. No, it is a ConditionalTheory not fact, so either it can take a MemberData as a second parameter, or can have InlineData attributes. It has the same behavior as [Theory] but with the condition.